### PR TITLE
HW1-Yi Huang

### PR DESCRIPTION
**Pull request by yihuang97**
I redid the homework and added a picture successfully this time!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Introduction of Yi Huang
+
+My name is Yi Huang, and I am originally from Shenzhen, China. I recently graduated from the University of Illinois at Urbana-Champaign, majoring in *psychology and economics*. **I had no experience in using R or Rmarkdown, but this homework is not as hard as I thought. The steps and instructions on the website are clear and straightforward.** My github homepage is https://github.com/yihuang97.
```
Comment (russell-roberts) [inline README.md:5]: One way to make the link cleaner is to do this:
[My Github Homepage](your url)

That way the url is not displayed, but the text itself serves as the link.

Comment (russell-roberts) [inline README.md:5]: Congratulations on your graduation!

**Review by russell-roberts — COMMENTED** [review]
Hi Yi, 

Nice use of text features (bold, italics)!  

When I viewed your document, the photo appeared as a broken link.  I'm not sure how to fix this, but I don't think it appears as you wanted it to. 

I also included a comment about how to include a cleaner link to your Github page.

Looks good!

**Review by hujinhui96 — COMMENTED** [review]
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | x |   |  
Coding strategy | x |   |  
Presentation | x |   |  
Achievement, creativity | x |   |  
Ease of access | x |   |  

Good bio overall. maybe have a picture in it?

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |         x     |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

Remarks:
1. You have experimented with 4 aspects of markdown language. Nicely done!
1. The link you put in the introduction could have been embedded in the text.
1. Your commit history is clear and helpful. Great job!

**Pull request closed without merging**